src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -54,7 +54,7 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $realCacheDir = $this->getContainer()->getParameter('kernel.cache_dir');
-        $oldCacheDir = $realCacheDir.'_old';
+        $oldCacheDir = substr($realCacheDir, 0, -1).'_';
```

Comment:
This will break if the real cache dir is already suffixed with `_`. So we need to make sure it's really different.